### PR TITLE
Omit option.

### PR DESCRIPTION
**Pull request by flamingbear**
**Summary:** 

Use destructuring to remove the keys from the existing granule.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,6 +75,7 @@
   "devDependencies": {
     "@ava/typescript": "^1.1.1",
     "@babel/core": "^7.9.6",
+    "@babel/plugin-proposal-object-rest-spread": "^7.15.6",
```
Comment (Jkovarik) [inline package.json:78]: Why is this needed?   Doesn't node 12 support it?

File: packages/api/.babelrc [hunk 2]
```diff
@@ -4,11 +4,12 @@
   "presets": [
     ["env", {
       "targets": {
-        "node": "10.16.3"
+        "node": "12.18.0"
```
Comment (Jkovarik) [inline packages/api/.babelrc:7]: 👍🏻

File: package.json (hunk 1 above)
Comment (Jkovarik) [reply package.json:78]: (the answer seems obvious, just making sure)

Comment (Jkovarik) [reply package.json:78]: This seems to work in the node 12 repl....

```
> foo = { alpha: 'foo', beta: 'bar', gamma: 'foobar' }
{ alpha: 'foo', beta: 'bar', gamma: 'foobar' }
> { alpha, ...beta } = foo
{ alpha: 'foo', beta: 'bar', gamma: 'foobar' }
> beta
{ beta: 'bar', gamma: 'foobar' }
>
```

File: packages/api/endpoints/granules.js [hunk 3]
```diff
@@ -136,13 +136,10 @@ const update = async (req, res) => {
...
-  const now = Date.now();
-  const updatedGranule = {
-    ...existingGranule,
-    updatedAt: now,
-    timestamp: now,
-    ...updatedBody,
-  };
+
+  // eslint-disable-next-line no-unused-vars
+  const { updatedAt, timestamp, ...existingGranuleWithoutTimes } = existingGranule;
```
Comment (Jkovarik) [inline packages/api/endpoints/granules.js:141]: Hmm, re: telling lint to ignore this, it favors terse over linting rules, and that one has somewhat good reasons for existing.  

If the goal here is to just remove timestamps from the object, why not just use our codebase's commonly-used `omit` pattern instead of a babel plugin and a lint override?

Comment (flamingbear) [reply packages/api/endpoints/granules.js:141]: haven't run into the omit pattern, and this was what made sense. That aside, the destructuring assignment is a useful tool to have even if you don't like this particular use of it.
